cfme/tests/infrastructure/test_vm_migrate.py

```diff
@@ -24,10 +24,10 @@ def test_vm_migrate(setup_provider, provider, request):
     Metadata:
         test_flag: migrate, provision
     """
-    vm = VM.factory("vmtest", provider)
+    vm = VM.factory("auto_test_services", provider)
     vm.migrate_vm("email@xyz.com", "first", "last")
     flash.assert_no_errors()
-    row_description = 'vmtest'
+    row_description = 'auto_test_services'
```

Comment:
Need an already existing VM to migrate so the vm should not be eliminated.
I am using auto_test_services for few other tests .